### ISSUE TITLE
The blade helper is one not the same with previous directive for @canImpersonate

### ISSUE THREAD

**Issue by rognales** [404labfr/laravel-impersonate#87]

**Comment by rognales** [comment]
Previous 
````php
if (app()['auth']->guard('$guard')->check()	
 && app()['auth']->guard('$guard')->user()->canImpersonate())
````

Now

````
return app('auth')->guard($guard)->check() && app('auth')->guard($guard)->canImpersonate();
````

This will trigger **BadMethodCallException**
````
Method Lab404\Impersonate\Guard\SessionGuard::canImpersonate does not exist.
````

_Workaround_
return app('auth')->guard($guard)->check() 
&& app('auth')->guard($guard)->**user()**->canImpersonate();

**Comment by rognales** [comment]
@MarceauKa

**Comment by MarceauKa** [comment]
You're right! I'm on it.

**Comment by MarceauKa** [comment]
The entire BladeDirectivesTest was malfunctionning due to a Laravel core change.
Ok fixed in [1.4.3](https://github.com/404labfr/laravel-impersonate/releases/tag/1.4.3).
